src/themes/theme.ts

```diff
@@ -5,6 +5,10 @@ export interface Theme {
     background?: ColorRepresentation;
     fog?: ColorRepresentation | null;
   };
+  /**
+   * Disable all activeFill, elements will not change color on hover/selection
+   */
+  disableActiveFill?: boolean;
```

Comment:
I don't love this being on the theme - what about passing it as a option to the selection hook?

Reply:
Got it - i think this is probably fine but i probably wouldn't have put it in the store personally just to avoid bloat but not a big deal.